packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue

```diff
@@ -0,0 +1,189 @@
+                :class="[$style['c-restaurantCard-rating-outOf']]"
+                :value="ratingsMax"
+                aria-hidden="true">
+                {{ ratingsMax }}
+            </data>
+
+            <!-- Number of ratings/Own rating message -->
+            <span aria-hidden="true">
+                &#40;
+                <data
+                    v-if="!isOwnRating"
+                    data-test-id="rating-count"
+                    :class="[$style['c-restaurantCard-rating-count']]"
+                    :value="count">
+                    {{ count }}
+                </data>
+                <span
+                    v-else
+                    data-test-id="rating-own-rating-message"
+                    :class="[$style['c-restaurantCard-rating-count']]">
+                    {{ isOwnRatingMessage }}
+                </span>
+                &#41;
+            </span>
+        </template>
+    </p>
+</template>
+
+<script>
+import { StarEmptyIcon, StarFilledIcon } from '@justeat/f-vue-icons';
+
+const RATINGS_MAX = 6;
+
+export default {
+    name: 'RestaurantRating',
+    components: {
+        StarEmptyIcon,
+        StarFilledIcon
+    },
+    props: {
+        /**
+         * Indications whether or not a user themselves has provided a rating for the item in question
+         */
+        isOwnRating: {
+            type: Boolean,
+            default: false
+        },
+        /**
+         * The mean value of the rating
+         */
+        mean: {
+            type: Number,
+            default: 0
+        },
+        /**
+         * How many ratings have been given for the item in question
+         */
+        count: {
+            type: Number,
+            default: 0
+        },
+        /**
+         * Message to display when no ratings available
+         * this is temporary whilst i18n is being fixed - ticket exists to fix
+         */
+        notRatedMessage: {
+            type: String,
+            default: ''
+        },
+        /**
+         * A visually hidden message to provide for screen readers that summarises the rating
+         * this is temporary whilst i18n is being fixed - ticket exists to fix
+         */
+        accessibleMessage: {
+            type: String,
+            default: ''
+        },
+        /**
+         * Message to display when `isOwnRating` is true
+         * this is temporary whilst i18n is being fixed - ticket exists to fix
+         */
+        isOwnRatingMessage: {
+            type: String,
+            default: ''
+        }
+    },
+    data () {
+        return {
+            ratingsMax: RATINGS_MAX
+        };
+    },
+    computed: {
+        noRatingsAvailable () {
+            return !this.count || !this.mean;
+        },
+        meanFormatted () {
+            return Number.parseFloat(this.mean).toFixed(2);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-restaurantCard-rating {
+    @include font-size($font-body-s-paragraph);
```

Comment:
You can use the alias keys (set in the [fozzie type map](https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L16-L54)) for the font-size mixin, so this would just be:

```suggestion
    @include font-size(body-s);
```

This by default also gives you the correct lint-height as well for that font-size, but if you don't need that you can set it as:

```scss
    @include font-size(body-s, false);
```

Reply:
cheers!